spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/core/convert/ConverterAwareMappingDatastoreEntityConverterTests.java

```diff
@@ -77,58 +84,71 @@ public void readTest() {
 	@Test
 	public void readNullTest() {
 		byte[] bytes = { 1, 2, 3 };
-		Entity entity = Entity.newBuilder(this.datastore.newKeyFactory().setKind("aKind").newKey("1"))
+		Entity entity = getaEntityBuilder()
+				.set("durationField", "PT24H")
 				.set("stringField", new NullValue())
 				.set("boolField", true)
 				.set("doubleField", 3.1415D)
 				.set("longField", 123L)
 				.set("latLngField", LatLng.of(10, 20))
 				.set("timestampField", Timestamp.ofTimeSecondsAndNanos(30, 40))
 				.set("blobField", Blob.copyFrom(bytes))
+				.set("intField", 99)
 				.build();
-		DatastoreEntityConverter entityConverter = new DatastoreEntityConverterImpl(new DatastoreMappingContext());
+		DatastoreEntityConverter entityConverter =
+				new ConverterAwareMappingDatastoreEntityConverter(new DatastoreMappingContext());
 		TestDatastoreItem item = entityConverter.read(TestDatastoreItem.class, entity);
 
 		assertThat(item.getStringField()).as("validate null field").isNull();
 	}
 
 	@Test(expected = DatastoreDataException.class)
 	public void testWrongTypeReadException() {
-				Entity entity = Entity.newBuilder(this.datastore.newKeyFactory().setKind("aKind").newKey("1"))
+		Entity entity = getaEntityBuilder()
```

Comment:
I think this needs to be "getAnEntity..." or "getOneEntity..." or just "getEntity..."

Reply:
It actually returns a builder, so renaming it to getEntity might be misleading

Reply:
Ah ok, then "getAnEntity" or "getOneEntity" or "get1Entity". It's just that "getaEntity" isn't correct because the "a" isn't capitalized and also "entity" starts with a vowel so it must be preceded by "an"

Reply:
oh, you right 
that was an IDE automatic refactoring generated name :)